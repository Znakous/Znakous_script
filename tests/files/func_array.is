f = [function(n) return 34 endfunction, 
    function(n) return n * 2 endfunction, 
    function() return [
        function(n) return n + 1 endfunction,
        function(n) return n + 2 endfunction,
        function(n) return [n, n * 10] endfunction
    ] endfunction, 
    function(u) {
        if u % 2 == 0 
            return "even"
        else
            return "odd"
        endif
    }
]

println(f[0](17), " ", f[2]()[1](10), " ", f[2]()[2](15)[1])
println((f[3](68) + " aboba ") * 2);
