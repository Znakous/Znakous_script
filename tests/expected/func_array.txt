34 12 150
even aboba even aboba 
